google-cloud-translate/src/main/java/com/google/cloud/translate/spi/v2/HttpTranslateRpc.java

```diff
@@ -126,6 +126,7 @@ private GenericUrl buildTargetUrl(String path) {
               .setSource(sourceLanguage)
               .setKey(options.getApiKey())
               .set("model", Option.MODEL.getString(optionMap))
```

Comment:
I don't know personally - my best guess is that the code was written against a version of the generated translate library before `setModel` was present, but when it was supported on the server side. It seems fine to use `setModel` now.